### PR TITLE
[13.0][ADD]stock_picking_orig_dest_link

### PR DESCRIPTION
**Pull request by GuillemCForgeFlow**
Adds the link between the origin and the destination pickings.
cc @ForgeFlow

### CODE REVIEW

**Review by JordiMForgeFlow — APPROVED** [review]
Functional and technical review 👍🏼

**Review by JaumeBforgeFlow — APPROVED** [review]
LGTM!

File: stock_picking_orig_dest_link/views/stock_picking_views.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                        domain="[]"
+                        context="{'group_by': 'orig_picking_ids'}"
+                    />
+                    <filter
+                        string="Destination Transfer/s"
+                        name="dest_picking_ids"
+                        domain="[]"
+                        context="{'group_by': 'dest_picking_ids'}"
+                    />
+                </filter>
```
Comment (LoisRForgeFlow) [inline stock_picking_orig_dest_link/views/stock_picking_views.xml]: I think you can remove this filter as the fields are m2m

File: stock_picking_orig_dest_link/readme/ROADMAP.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* This module is very similar to *stock_picking_show_linked* in terms of funcionalities being added. In future versions, we should consider merging the two modules into a single one.
```
Comment (LoisRForgeFlow) [inline stock_picking_orig_dest_link/readme/ROADMAP.rst:1]: :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM code review

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM code review

**Pull request merged**